test/fixtures/smart_collection_products.json

```diff
@@ -0,0 +1,153 @@
+          "position": 1,
+          "price": "199.00",
+          "product_id": 632910392,
+          "created_at": "2011-10-20T14:05:13-04:00",
+          "requires_shipping": true,
+          "title": "Pink",
+          "inventory_quantity": 10,
+          "compare_at_price": null,
+          "inventory_policy": "continue",
+          "updated_at": "2011-10-20T14:05:13-04:00",
+          "inventory_management": "shopify",
+          "id": 808950810,
+          "taxable": true,
+          "grams": 200,
+          "sku": "IPOD2008PINK",
+          "option1": "Pink",
+          "fulfillment_service": "manual",
+          "option2": null,
+          "option3": null
+        },
+        {
+          "position": 2,
+          "price": "199.00",
+          "product_id": 632910392,
+          "created_at": "2011-10-20T14:05:13-04:00",
+          "requires_shipping": true,
+          "title": "Red",
+          "inventory_quantity": 20,
+          "compare_at_price": null,
+          "inventory_policy": "continue",
+          "updated_at": "2011-10-20T14:05:13-04:00",
+          "inventory_management": "shopify",
+          "id": 49148385,
+          "taxable": true,
+          "grams": 200,
+          "sku": "IPOD2008RED",
+          "option1": "Red",
+          "fulfillment_service": "manual",
+          "option2": null,
+          "option3": null
+        },
+        {
+          "position": 3,
+          "price": "199.00",
+          "product_id": 632910392,
+          "created_at": "2011-10-20T14:05:13-04:00",
+          "requires_shipping": true,
+          "title": "Green",
+          "inventory_quantity": 30,
+          "compare_at_price": null,
+          "inventory_policy": "continue",
+          "updated_at": "2011-10-20T14:05:13-04:00",
+          "inventory_management": "shopify",
+          "id": 39072856,
+          "taxable": true,
+          "grams": 200,
+          "sku": "IPOD2008GREEN",
+          "option1": "Green",
+          "fulfillment_service": "manual",
+          "option2": null,
+          "option3": null
+        },
+        {
+          "position": 4,
+          "price": "199.00",
+          "product_id": 632910392,
+          "created_at": "2011-10-20T14:05:13-04:00",
+          "requires_shipping": true,
+          "title": "Black",
+          "inventory_quantity": 40,
+          "compare_at_price": null,
+          "inventory_policy": "continue",
+          "updated_at": "2011-10-20T14:05:13-04:00",
+          "inventory_management": "shopify",
+          "id": 457924702,
+          "taxable": true,
+          "grams": 200,
+          "sku": "IPOD2008BLACK",
+          "option1": "Black",
+          "fulfillment_service": "manual",
+          "option2": null,
+          "option3": null
+        }
+      ],
+      "vendor": "Apple",
+      "published_at": "2007-12-31T19:00:00-05:00",
+      "options": [
+        {
+          "name": "Title"
+        }
+      ]
+    },
+    {
+      "product_type": "Cult Products",
```

Comment:
are u gonna add `manually_sorted` to the products?

Reply:
I decided to add it here afterall